homeassistant/components/tractive/__init__.py

```diff
@@ -148,6 +148,13 @@ async def _generate_trackables(
         tracker.details(), tracker.hw_info(), tracker.pos_report()
     )
 
+    if not tracker_details.get("_id"):
+        _LOGGER.info(
+            "Tractive API returns incomplete data for tracker %s",
```

Comment:
Pass the message as an exception argument instead and the message will be logged at debug level by default.

Reply:
Ok, I'll prepare a PR tomorrow. Thanks.

Reply:
https://github.com/home-assistant/core/pull/118534